dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/security/ApiTokenController.java

```diff
@@ -81,15 +79,6 @@ public class ApiTokenController extends AbstractCrudController<ApiToken>
 
     private final ApiTokenService apiTokenService;
 
-    // Overwritten to get full access control on GET single object
```

Comment:
@netroms I concluded that this comment (and the override) must be outdated since it is basically a copy of the overridden implementation. Or I fail to see how overriding the method with same implementation has any effect on access control. Please double check.